### PR TITLE
Rc5 migration

### PR DESCRIPTION
**Pull request by thuynh02**
Created app.module and updated some syntax to account for migration from rc4 to rc5. A later pull request will be made to remove commented out sections (like the old method of bootstrapping and unused script tags in index.html)

### CODE REVIEW

File: src/app/api-caller/api-caller.component.ts [hunk 1]
```diff
@@ -16,7 +12,7 @@ let imgBackground = require('../img/api-caller-bg.jpg');
   selector: 'app-api-caller',
   templateUrl: 'api-caller.component.html',
   styleUrls: ['api-caller.component.scss'],
-  directives: [ MdButton, MdCard, MdInput]
+  directives: []
```
Comment (ironmanwaring) [inline src/app/api-caller/api-caller.component.ts]: i don't think you need the directives property anymore with rc5

File: src/app/app.component.ts [hunk 2]
```diff
@@ -1,23 +1,11 @@
...
   templateUrl: 'app.component.html',
   styleUrls: ['app.component.scss', '../custom-lock.css'],
-  directives: [
-    MD_RIPPLE_DIRECTIVES,
-    ROUTER_DIRECTIVES,
-    MdToolbar,
-    MdButton
-  ],
-  providers: [ AuthService ]
+  providers: []
```
Comment (ironmanwaring) [inline src/app/app.component.ts]: i don't think you need providers anymore, either

File: src/app/home/home.component.ts [hunk 3]
```diff
@@ -17,7 +13,7 @@ let imgBecomeCoach = require('../img/become-coach.jpg');
   selector: 'app-home',
   templateUrl: 'home.component.html',
   styleUrls: ['home.component.scss'],
-  directives: [MdCard, MdButton]
+  directives: []
```
Comment (ironmanwaring) [inline src/app/home/home.component.ts]: still need this property?

**Comment by ironmanwaring** [comment]
other than a few properties in component decorators we may not need lgtm

**Comment by ironmanwaring** [comment]
@thuynh02 what's the status of this pr?

**Comment by thuynh02** [comment]
Ah! I think these comments got lost in the mix of other github emails. The base changes are present in  pull request #24 and I can remove those extraneous directive/provider attributes in component. This can be closed.

**Pull request closed without merging**